pkg/aspect/accessLogsManager.go

```diff
@@ -32,28 +33,54 @@ type (
 	accessLogsManager struct{}
 
 	accessLogsWrapper struct {
-		logName   string
-		inputs    map[string]string // map from param to expr
-		aspect    adapter.AccessLogsAspect
-		attrNames []string
-		template  *template.Template
+		name          string
+		aspect        adapter.AccessLogsAspect
+		labels        map[string]string // label name -> expression
+		template      *template.Template
+		templateExprs map[string]string // template variable -> expression
 	}
 )
 
 const (
 	// TODO: revisit when well-known attributes are defined.
 	commonLogFormat = `{{or (.originIp) "-"}} - {{or (.source_user) "-"}} ` +
-		`[{{or (.timestamp.Format "02/Jan/2006:15:04:05 -0700") "-"}}] "{{or (.apiMethod) "-"}} ` +
+		`[{{or (.timestamp.Format "02/Jan/2006:15:04:05 -0700") "-"}}] "{{or (.method) "-"}} ` +
 		`{{or (.url) "-"}} {{or (.protocol) "-"}}" {{or (.responseCode) "-"}} {{or (.responseSize) "-"}}`
 	// TODO: revisit when well-known attributes are defined.
 	combinedLogFormat = commonLogFormat + ` "{{or (.referer) "-"}}" "{{or (.user_agent) "-"}}"`
 )
 
 var (
+	commonLogTemplateExpressions = map[string]string{
```

Comment:
I really don't like having attribute names baked into our code like that... I'd rather we leave the default blank and require the operator to supply a default block of config that produces the same thing.

In other words, we could only support the custom log format and require the operator configure things.